client/src/components/App.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Commentbar from './Commentbar.jsx';
+import ArtistInfo from './ArtistInfo.jsx';
+import TrackInfo from './TrackInfo.jsx';
+import Comments from './Comments';
+import './styles.css';
+
+class App extends React.Component {
+    constructor(props){
+        super(props);
+        this.state = {
+            comments: [],
+        }
+    };
+
+
+    handleAdd(newComment){
```

Comment:
I take it this is a function which is not in use yet? I don't see it referenced anywhere.

Reply:
Not in use yet, will make sure it's implemented in final code.